### PR TITLE
Add LIN and FlexRay status channel tests.

### PR DESCRIPTION
**Pull request by douglasnorman**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-xnet-system-tests/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

It adds new tests for LIN and FlexRay status channels.

### Why should this Pull Request be merged?

This is so LIN and FlexRay status channels are automatically tested.

### What testing has been done?

All VI tests were successfully run.

### CODE REVIEW

**Comment by douglasnorman** [comment]
I have divided things up such that each comparison is checked individually, and has a specific fail message.

File: Source/Tests/LIN Loopback/Assets/LIN Unconditional Loopback Disabled with Status Channels.nivssdf [hunk 1]
```diff
@@ -0,0 +1,1976 @@
...
+										<Property Name="user.CD.Cluster name">
+											<String>Cluster</String>
+										</Property>
+										<Property Name="user.CD.LIN schedules">
+											<String>FastSchedule;MasterRequest_Schedule;SlaveResponse_Schedule;SlowSchedule
+</String>
+										</Property>
+										<Property Name="user.CD.Master">
+											<Boolean>true</Boolean>
+										</Property>
```
Comment (rtzoeller) [inline Source/Tests/LIN Loopback/Assets/LIN Unconditional Loopback Disabled with Status Channels.nivssdf:191]: @Karl-G1 you mentioned that you tested these, but I thought we had previously run into issues with the master port not being the outgoing port.

Was that only on PharLap? Should the outgoing port be the master instead?

Comment (Karl-G1) [reply Source/Tests/LIN Loopback/Assets/LIN Unconditional Loopback Disabled with Status Channels.nivssdf:191]: Ah, I once again tested these on Linux PXI (it's the last target used in the test run).  I did not check on Pharlap, and the Outgoing port should be the LIN Master as we saw in PR #9.

Comment (rtzoeller) [reply Source/Tests/LIN Loopback/Assets/LIN Unconditional Loopback Disabled with Status Channels.nivssdf:191]: @douglasnorman can you make sure this gets updated in all LIN system definitions?

**Comment by rtzoeller** [comment]
@douglasnorman you should pass the custom failure messages to the `message` terminal, rather than the `custom report message` terminal. 

It's not clear from the documentation, but `custom report message` _always_ gets added to the test output, whereas `message` only gets added on failure (which is what we want).

**Comment by douglasnorman** [comment]
I changed the output ports to Master, and I wired failure text to "message:.

**Pull request merged**